tensorflow/lite/toco/graph_transformations/quantize.cc

```diff
@@ -62,8 +62,9 @@ bool SupportsQuantization(const Operator& op) {
          type == OperatorType::kLessEqual || type == OperatorType::kSelect ||
          type == OperatorType::kArgMax || type == OperatorType::kRelu ||
          type == OperatorType::kRelu1 || type == OperatorType::kRelu6 ||
-         type == OperatorType::kShape || type == OperatorType::kExpandDims ||
-         type == OperatorType::kPack || type == OperatorType::kTopK_V2 ||
+         type == OperatorType::kLeakyRelu || type == OperatorType::kShape ||
```

Comment:
This is not related to the strided slice, does it ?

Reply:
@karimnosseir , thanks or the review, i have removed this part.

Regards
Amit